src/gatsby-scaffolder.js

```diff
@@ -0,0 +1,35 @@
+import {promises as fs} from 'fs';
+import {resolve} from 'path';
+import mkdir from '../thirdparty-wrappers/make-dir';
+
+export default async function ({projectRoot}) {
+  const pathToPagesDirectory = await mkdir(`${projectRoot}/src/pages`);
+  await Promise.all([
+    fs.copyFile(resolve(__dirname, '..', 'templates', 'gatsby-config.js'), `${projectRoot}/gatsby-config.js`),
```

Comment:
i have been debating pulling some of these details up to `@form8ion/core` lately. i don't necessarily dislike using `fs` raw like this, but reinforcing the conventions around where the `templates/` are located make it potentially seem worth it.

there are a few of the more complicated scaffolders where i've had to be more careful of resolving the `templates/` since the src vs bundled file structures are different. doesnt happen all that often, but that complexity could be hidden away and only used when needed.

anyway, not sure i'll get around to something like this any time soon, but figured i should share since it's at least on my mind lately